src/components/ProofCard.vue

```diff
@@ -50,6 +50,10 @@ export default {
       type: String,
       default: '100%',
     },
+    showThumbImage: {
```

Comment:
the backend field name is `image_thumb_path` so I'd rather have the same order for the prop for clarity :)

```suggestion
    showImageThumb: {
```